modules/core/src/main/java/org/apache/ignite/maintenance/MaintenanceRegistry.java

```diff
@@ -97,7 +98,28 @@
      * @return Previously registered {@link MaintenanceTask} with the same ID
      * or null if no tasks were registered for this ID.
      */
-    public @Nullable MaintenanceTask registerMaintenanceTask(MaintenanceTask task) throws IgniteCheckedException;
+    @Nullable public MaintenanceTask registerMaintenanceTask(MaintenanceTask task) throws IgniteCheckedException;
```

Comment:
```suggestion
    @Nullable MaintenanceTask registerMaintenanceTask(MaintenanceTask task) throws IgniteCheckedException;
```

Reply:
`Public` was there already, I'm not sure why we should remove it

Reply:
This is the interface, it is by default, you can leave it.